src/Tools/dotnet-dev-certs/src/Program.cs

```diff
@@ -157,7 +158,16 @@ private static int CheckHttpsCertificate(CommandOption trust, IReporter reporter
             }
             else
             {
-                reporter.Verbose("A valid certificate was found.");
+                if (RuntimeInformation.IsOSPlatform(OSPlatform.OSX) && certificateManager.HasValidCertificateWithInnaccessibleKeyAcrossPartitions())
+                {
+                    reporter.Warn($"We found a valid HTTPS certificate but we couldn't determine that the key will be accessible across security partitions. Running dotnet dev-certs https will fix the issue.");
```

Comment:
```suggestion
                    reporter.Warn($"We found a valid HTTPS certificate but we couldn't confirm that the key will be accessible across security partitions. Run dotnet dev-certs https to ensure it will be accessible during development.");
```